app/components/button.jsx

```diff
@@ -1,4 +1,5 @@
 // https://github.com/EnCiv/civil-pursuit/issues/43
+//https://github.com/EnCiv/civil-pursuit/issues/257
```

Comment:
since this file has not changed, please revert it back to what's in master.
git checkout master app/components/button.jsx